spec/spec_helper.rb

```diff
@@ -111,6 +111,12 @@
     expectations.syntax = :expect
   end
 
+  # Precompile Webpacker assets (once) when starting the suite. The default setup can result
+  # in the assets getting compiled many times throughout the build, slowing it down.
+  config.before :suite do
+    Webpacker.compile
```

Comment:
The build time for jobs in the test suite running feature specs basically _doubled_ until I figured out a solution here, then they went back to normal. I think the assets were being precompiled again on every single request :scream: 

This corresponds with adjusting the defaults in `config/webpacker.yml` for the test environment so it uses production-like settings with precompiling and caching (compile once at the start, and don't recompile if the assets are already present).